cli/cdsctl/config.go

```diff
@@ -60,74 +58,87 @@ func loadConfig(cmd *cobra.Command) (*cdsclient.Config, error) {
 		}
 	}
 
-	dir, err := os.Getwd()
-	if err != nil {
-		return nil, err
+	if configFile == "" {
+		configFile = path.Join(userHomeDir(), ".cdsrc")
 	}
 
-	homedir := userHomeDir()
-
-	var configFiles []string
-	if configFile != "" {
-		configFiles = []string{configFile}
-	} else {
-		configFiles = []string{
-			path.Join(dir, ".cdsrc"),
-			path.Join(homedir, ".cdsrc"),
-		}
+	if verbose {
+		fmt.Println("Configuration loaded from", configFile)
 	}
 
-	var i int
-	for c.Host == "" && i < len(configFiles) {
-		if _, err := os.Stat(configFiles[i]); err == nil {
-			f, err := os.Open(configFiles[i])
-			if err != nil {
-				if verbose {
-					fmt.Printf("Unable to read %s \n", configFiles[i])
-				}
-				return nil, err
+	cdsContext := &internal.CDSContext{}
+	if _, err := os.Stat(configFile); err == nil {
+		f, err := os.Open(configFile)
+		if err != nil {
+			if verbose {
+				fmt.Printf("Unable to read %s \n", configFile)
 			}
-			defer f.Close()
+			return "", nil, err
```

Comment:
remove the verbose and do:
`return "", nil, fmt.Errorf("unable to read file %s: %v", configFile, err)`